bin/deploy

```diff
@@ -91,6 +91,9 @@ then
 fi
 git checkout -b $DEPLOY_BRANCH
 
+# Export JEKYLL_ENV=production
+export JEKYLL_ENV=production
```

Comment:
why is this necessary?

Reply:
`jekyll-minifier` only works when `JEKYLL_ENV=production` is set.

This tells the plugin that this site is build for production, that way this won't work during testing but will work while deploying. This will save a lot of time during testing, as it takes a bit of time to minify all the files.